From f48acc22f0105ce6bac63294d248873ae231c6cc Mon Sep 17 00:00:00 2001
From: Luca Beltrame <lbeltrame@kde.org>
Date: Tue, 28 Jun 2016 00:56:11 +0200
Subject: [PATCH] Remove (improper?) KSharedConfigPtr usage

1f47df5074e041cad803863e5ef04cb082faf601 in kconfig removed
KSharedConfigPtr. Use KSharedConfig::Ptr instead, as used in kconfig
itself.
---
 ktorrent/dialogs/pastedialog.cpp  | 4 ++--
 ktorrent/dialogs/pastedialog.h    | 5 +++--
 plugins/mediaplayer/mediaview.cpp | 4 ++--
 plugins/mediaplayer/mediaview.h   | 6 +++---
 4 files changed, 10 insertions(+), 9 deletions(-)

diff --git a/ktorrent/dialogs/pastedialog.cpp b/ktorrent/dialogs/pastedialog.cpp
index d0f08e9..4708bca 100644
--- a/ktorrent/dialogs/pastedialog.cpp
+++ b/ktorrent/dialogs/pastedialog.cpp
@@ -73,14 +73,14 @@ namespace kt
         m_groups->addItems(grps);
     }
 
-    void PasteDialog::loadState(KSharedConfigPtr cfg)
+    void PasteDialog::loadState(KSharedConfig::Ptr cfg)
     {
         KConfigGroup g = cfg->group("PasteDlg");
         m_silently->setChecked(g.readEntry("silently", false));
         m_groups->setCurrentIndex(g.readEntry("group", 0));
     }
 
-    void PasteDialog::saveState(KSharedConfigPtr cfg)
+    void PasteDialog::saveState(KSharedConfig::Ptr cfg)
     {
         KConfigGroup g = cfg->group("PasteDlg");
         g.writeEntry("silently", m_silently->isChecked());
diff --git a/ktorrent/dialogs/pastedialog.h b/ktorrent/dialogs/pastedialog.h
index 2446848..0c9a65b 100644
--- a/ktorrent/dialogs/pastedialog.h
+++ b/ktorrent/dialogs/pastedialog.h
@@ -22,6 +22,7 @@
 #define PASTEDIALOG_H
 
 #include <KDialog>
+#include <KSharedConfig>
 #include "ui_pastedlgbase.h"
 
 
@@ -43,12 +44,12 @@ namespace kt
         /**
         * Load the state of the dialog
         */
-        void loadState(KSharedConfigPtr cfg);
+        void loadState(KSharedConfig::Ptr cfg);
 
         /**
         * Save the state of the dialog
         */
-        void saveState(KSharedConfigPtr cfg);
+        void saveState(KSharedConfig::Ptr cfg);
 
     public slots:
         virtual void accept();
diff --git a/plugins/mediaplayer/mediaview.cpp b/plugins/mediaplayer/mediaview.cpp
index d805c4b..0cbd901 100644
--- a/plugins/mediaplayer/mediaview.cpp
+++ b/plugins/mediaplayer/mediaview.cpp
@@ -146,7 +146,7 @@ namespace kt
         filter->setShowIncomplete(on);
     }
 
-    void MediaView::loadState(KSharedConfigPtr cfg)
+    void MediaView::loadState(KSharedConfig::Ptr cfg)
     {
         KConfigGroup g = cfg->group("MediaView");
         show_incomplete->setChecked(g.readEntry("show_incomplete", false));
@@ -154,7 +154,7 @@ namespace kt
 
     }
 
-    void MediaView::saveState(KSharedConfigPtr cfg)
+    void MediaView::saveState(KSharedConfig::Ptr cfg)
     {
         KConfigGroup g = cfg->group("MediaView");
         g.writeEntry("show_incomplete", show_incomplete->isChecked());
diff --git a/plugins/mediaplayer/mediaview.h b/plugins/mediaplayer/mediaview.h
index 2b9a7e6..ba2e027 100644
--- a/plugins/mediaplayer/mediaview.h
+++ b/plugins/mediaplayer/mediaview.h
@@ -25,7 +25,7 @@
 #include <QListView>
 #include <QCheckBox>
 #include <QSortFilterProxyModel>
-#include <KSharedConfigPtr>
+#include <KSharedConfig>
 #include "mediafile.h"
 
 
@@ -68,8 +68,8 @@ namespace kt
         MediaView(MediaModel* model, QWidget* parent);
         virtual ~MediaView();
 
-        void saveState(KSharedConfigPtr cfg);
-        void loadState(KSharedConfigPtr cfg);
+        void saveState(KSharedConfig::Ptr cfg);
+        void loadState(KSharedConfig::Ptr cfg);
 
     signals:
         void doubleClicked(const MediaFileRef& mf);
